src/test/java/io/apicurio/datamodels/refs/DereferenceTestCase.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 JBoss Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.datamodels.refs;
+
+/**
+ * @author eric.wittmann@gmail.com
+ */
+public class DereferenceTestCase {
```

Comment:
Am I the only one who loves Lombok? :)

Reply:
I'm coming around to it, but IDEs are also pretty good at just adding the shit you need.  :)